edx_credentials_themes/static/edx.org/certificates/sass/_layouts-rendering.scss

```diff
@@ -26,7 +26,6 @@
       position: absolute;
       right: 375px;
       top: 0;
-      opacity: 0.5;
```

Comment:
Change here is because the old watermark was darker. This one doesn't need the opacity adjustment